packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md

```diff
@@ -0,0 +1,377 @@
+
+</div>
+
+## Register an Okta Aerial API client
+
+Use an Aerial API client to add orgs to the Aerial account and to modify products.
+
+Only a super admin in your Aerial org can grant scopes to the client. Configure scopes once per API client.
+
+In your Aerial org, create an API client:
+
+1. In the **Admin Console**, go to **Applications** > **Applications**, and then click **Create App Integration**.
+1. Select **API Services** as the sign-in method, and click **Next**.
+1. Enter a name for your client and click **Save**.
+
+Configure the signing keys for the client:
+
+1. In the **Client Credentials** section of the **General** tab, click **Edit** to change the client authentication method.
+1. Select **Public key/Private key** as the client authentication method.
+1. Choose either **Save keys in Okta** or **Use a URL to fetch keys dynamically**:
+   - If you want to save keys in Okta, click **Add key**.
+   - If you want to use a URL to fetch keys dynamically, you need to provide a URL that returns the [JWKS documents](https://www.rfc-editor.org/rfc/rfc7517#section-5).
+
+Grant scopes to the client:
+
+1. Select the **Okta API Scopes** tab.
+1. To access Okta Aerial, click **Grant** on the following scopes:
+  - `okta.accounts.manage`: read/write operations
+  - `okta.accounts.read`: read-only operations
+
+## Authenticate with Okta Aerial
+
+To authenticate with Okta Aerial, a client obtains an access token from the Aerial org. See [Implement OAuth for Okta with a service app](/docs/guides/implement-oauth-for-okta-serviceapp/main/#get-an-access-token) for more information.
+
+Create a [JWT assertion](/docs/guides/implement-oauth-for-okta-serviceapp/main/#create-and-sign-the-jwt) and use it to make a [token request](/docs/guides/implement-oauth-for-okta-serviceapp/main/#create-and-sign-the-jwt) to the Aerial org. The Aerial org returns the access token:
+
+```bash
+curl --location --request POST 'https://${aerialOrgDomain}/oauth2/v1/token' \
+    -H 'Accept: application/json' \
+    -H 'Content-Type: application/x-www-form-urlencoded' \
+    --data-urlencode 'grant_type=client_credentials' \
+    --data-urlencode 'scope=okta.accounts.manage' \
+    --data-urlencode 'client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer' \
+    --data-urlencode 'client_assertion=${jwt_assertion}'
+```
+
+Add the access token to the Authorization header of Okta Aerial API requests:
+
+```bash
+Authorization: Bearer ${access_token}
+```
+<!-- our OAuth docs for service apps rely on Postman for this step. need something in the interim until Postman is ready -->
+
+## Create a child Org
+
+Create a child org using the Org creator API credentials in the parent org. This creates a child org with features synced from the parent org. In the API response, you receive an API token tied to the super admin. Use the token to provision more resources on the child org, like policies, apps, or groups.
```

Comment:
This first sentence was confusing to me. Please LMK if I changed the meaning with my edit.

```suggestion
Create a child org of the parent org using the Org creator API. This creates a child org with features synced from the parent org. In the API response, you receive an API token tied to the super admin. Use the token to provision more resources on the child org like policies, apps, or groups.
```